src/opening-period/form/OpeningPeriodForm.tsx

```diff
@@ -193,8 +217,21 @@ export default function OpeningPeriodForm({
               onChange={(value): void => setPeriodEndDate(value || null)}
               value={periodEndDate}
               registerFn={register}
+              customValidations={{
+                dateRange: validateEndInputWithStartDate(periodBeginDate),
```

Comment:
Tested the feature locally and it looked nice! 👍  I wonder if we could/should trigger the validation after user has selected the dates? Currently the validation is triggered by user trying to send the form. I guess it could be ok to begin with, and then we could improve it later on.

Reply:
I think it is doable with relative small changes. I’ll test 👍